doc-content/enterprise-only/optimizer/quarkus-creating-proj-cli-proc.adoc

```diff
@@ -0,0 +1,191 @@
+[id='proc-quarkus-creating-proj-cli_{context}']
+
+= Creating a {QUARKUS} Maven project using the Quarkus CLI
+
+You can use the Quarkus command line interface (CLI) to create a Quarkus OptaPlanner project.
+
+.Prerequisites
+* You have installed the Quarkus CLI. For information, see https://quarkus.io/guides/cli-tooling[Building Quarkus Apps with Quarkus Command Line Interface].
+
+
+.Procedure
+
+. Create a Quarkus application:
++
+[source]
+----
+quarkus create app -P io.quarkus:quarkus-bom:{QUARKUS_BOM}
+----
+
+. To view the available extensions, enter the following command:
++
+[source]
+----
+quarkus ext -i
+----
++
+This command returns the following extensions:
++
+[source]
+----
+optaplanner-quarkus
+optaplanner-quarkus-benchmark
+optaplanner-quarkus-jackson
+optaplanner-quarkus-jsonb
+----
+
+. Enter the following command to add extensions to the project's `pom.xml` file:
++
+[source]
+----
+quarkus ext add resteasy-jackson
+quarkus ext add optaplanner-quarkus
+quarkus ext add optaplanner-quarkus-jackson
+----
+. Open the `pom.xml` file in a text editor. The contents of the file should look similar to the following example:
++
+[source]
+----
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.acme</groupId>
+  <artifactId>code-with-quarkus-optaplanner</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <properties>
+	<compiler-plugin.version>3.8.1</compiler-plugin.version>
+	<maven.compiler.parameters>true</maven.compiler.parameters>
+	<maven.compiler.source>11</maven.compiler.source>
+	<maven.compiler.target>11</maven.compiler.target>
+	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	<quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+	<quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+	<quarkus.platform.version>2.2.2.Final-redhat-0000x</quarkus.platform.version>
```

Comment:
This should be {QUARKUS_BOM}

Reply:
.Fixed.

Reply:
This {QUARKUS_BOM} would be substituted with the product version of Quarkus, right?

Reply:
Yes, currently 1.11.6.Final-redhat-00001. Please check the HTML linked above.

Reply:
LGTM